lib/manager/gradle/shallow/extract.spec.ts

```diff
@@ -136,4 +137,73 @@ describe('manager/gradle/shallow/extract', () => {
       },
     ]);
   });
+
+  it('works with dependency catalogs', async () => {
+    const fsMock = {
+      'gradle/libs.versions.toml': dedent`
+        [versions]
+        detekt = "1.17.0"
+        kotest = "4.6.0"
+        publish-on-central = "0.5.0"
+
+        [libraries]
+        detekt-formatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", version.ref = "detekt" }
```

Comment:
We should add another one: `alias = "artifact"`, i.e:

```toml
detekt-formatting = "io.gitlab.arturbosch.detekt:detekt-formatting:1.18.0" }
```

Reply:
There's a test now with this case